src/DashboardHub/Bundle/AppBundle/Service/GithubService.php

```diff
@@ -122,4 +122,39 @@ public function getBranches($reponame, $limit = 5)
 
         return $branches;
     }
+
+    /**
+     * @param string $reponame
+     *
+     * @return array $branches
+     */
+    public function getMilestones($reponame, $limit = 5)
+    {
+        $cache    = $this->cache->getItem(__METHOD__, $reponame, $limit);
+        $milestones = $cache->get();
+        if ($cache->isMiss()) {
+            $milestones = json_decode(
+                $this->client
+                    ->get('/repos/' . $reponame . '/milestones?per_page=' . $limit)
+                    ->getBody()
+                    ->getContents()
+                ,
+                true
+            );
+
+            // calculate Completeness percentage
+            foreach($milestones as $key => $milestone) {
+                if ($milestone['closed_issues'] + $milestone['open_issues'] != 0) {
+                    $milestones[ $key ]['completeness'] =
+                        round(($milestone['closed_issues'] / ($milestone['closed_issues'] + $milestone['open_issues'])) * 100);
```

Comment:
Sure. This will get refactored shortly to its own method, but will split this line for now.